applications/GeoMechanicsApplication/custom_elements/transient_Pw_line_element.h

```diff
@@ -47,8 +47,7 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) TransientPwLineElement : public Elem
                            std::unique_ptr<IntegrationCoefficientModifier> pCoefficientModifier)
         : Element(NewId, pGeometry),
           mContributions(rContributions),
-          mpIntegrationCoefficientsCalculator{
-              std::make_unique<IntegrationCoefficientsCalculator>(std::move(pCoefficientModifier))}
+          mIntegrationCoefficientsCalculator{IntegrationCoefficientsCalculator(std::move(pCoefficientModifier))}
```

Comment:
```suggestion
          mIntegrationCoefficientsCalculator{std::move(pCoefficientModifier)}
```

This suggestion also applies to line 61.

Reply:
done